ignite/metrics/root_mean_squared_error.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import division
+import math
+
+import torch
+
+from .metric import Metric
+from ignite.exceptions import NotComputableError
+
+
+class RootMeanSquaredError(Metric):
```

Comment:
could this not inherit from MeanSquaredError and just override the `compute` function to be:
```python
def compute(self):
    mse = super(self, RootMeanSquaredError).compute()
    return math.sqrt(mse)
```

Reply:
Definitely. One other option is to just have MeanSquaredError but have it take a `root` flag in the constructor. I think I prefer the more explicit approach of having both MeanSquaredError and RootMeanSquaredError though. What do you think?

Reply:
@alykhantejani I went ahead and made your suggested change. I can definitely update it though if you wanna go a different route.

Reply:
👍 I also like the explicit route